tests/h/api/search/index_test.py

```diff
@@ -111,6 +111,31 @@ def log(self, patch):
         return patch('h.api.search.index.log')
 
 
+@pytest.mark.usefixtures('BatchIndexer', 'BatchDeleter')
+class TestReindex(object):
+    def test_it_indexes_all_annotations(self, BatchIndexer):
+        session, es, request = mock.Mock(), mock.Mock(), mock.Mock()
```

Comment:
For situations like this where you're just passing named things around, it's probably nicer to use `mock.sentinel`:

```python
s = mock.sentinel
index.reindex(s.session, s.es, s.request)
BatchIndexer.assert_called_once_with(s.session, s.es, s.request)
```